ignite/templates/app/stargate/app/app.go.plush

```diff
@@ -384,7 +387,7 @@ func New(
 
 	// Create static IBC router, add transfer route, then set and seal it
 	ibcRouter := ibcporttypes.NewRouter()
```

Comment:
Sorry isn't should be `ibcRouter := porttypes.NewRouter()`

From: 
package:  porttypes ("github.com/cosmos/ibc-go/v3/modules/core/05-port/types")

Reply:
yes! is the same thing: https://github.com/ignite-hq/cli/pull/2566/files#diff-5152ca3df036ab6cd260e7fca13f76937aa7b276e1fd05c8879b2088ec338454R84